cmd/auth/describe.go

```diff
@@ -0,0 +1,180 @@
+package auth
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/databricks/cli/cmd/root"
+	"github.com/databricks/cli/libs/cmdio"
+	"github.com/databricks/cli/libs/flags"
+	"github.com/databricks/databricks-sdk-go/config"
+	"github.com/spf13/cobra"
+)
+
+var authTemplate = `{{"Workspace:" | bold}} {{.Details.Host}}
+{{if .Username }}{{"User:" | bold}} {{.Username}}{{end}}
+{{"Authenticated with:" | bold}} {{.Details.AuthType}}
```

Comment:
As a new user, it took me a hot second to figure out what 'pat' was in the example output, and I also wondered what other types there were, and had to do a bit of searching. Low priority, but just a thought - could we output a friendlier string instead of just the exact field value?

Reply:
If we do, we should do both (can be a follow-up to this, IMO). The literal value is what someone would use when hardcoding it in a profile or in the `DATABRICKS_AUTH_TYPE` environment variable.